Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -86,28 +86,9 @@ - (void)layoutSubviews {
     self.shapeLayer.frame = CGRectMake(paddingX, 0, BPKCalendarCellSpacing.cellCircleHeight, BPKCalendarCellSpacing.cellCircleHeight);
 
     switch (self.rowType) {
-    case RowTypeStart:
```

Comment:
While we are here can we change the initial value of `corners` from `0` to `UIRectCornerAllCorners`? That makes the code easier to read because you don't have to figure out what the enum value `0` means.

Getting rid of the RTL logic is a solid win